src/server/stream_family.cc

```diff
@@ -907,6 +907,144 @@ OpResult<pair<stream*, streamCG*>> FindGroup(const OpArgs& op_args, string_view
   return res;
 }
 
+constexpr uint8_t kClaimForce = 1 << 0;
+constexpr uint8_t kClaimJustID = 1 << 1;
+
+struct ClaimOpts {
+  string_view group;
+  string_view consumer;
+  int64 min_idle_time;
+  ParsedStreamId lastid;
```

Comment:
Do I understand correctly, that this is only to be used in the future?
As such, I'd remove it for now. No point in having dead code. The TODO below is sufficient.
Once you do that (in a future PR), wouldn't this be better as `streamID` rather than `ParsedStreamId`?